.github/workflows/ci.yml

```diff
@@ -173,17 +151,26 @@ jobs:
         image: rabbitmq:latest
         ports:
           - 5672:5672
-        options: --health-cmd "rabbitmqctl node_health_check" --health-interval 10s --health-timeout 5s --health-retries 5
+        options: >-
+          --health-cmd "rabbitmqctl node_health_check" 
+          --health-interval 10s 
+          --health-timeout 5s 
+          --health-retries 5
       memcached:
         image: memcached:latest
         ports:
           - 11211:11211
-        options: --health-cmd "timeout 5 bash -c 'cat < /dev/null > /dev/udp/127.0.0.1/11211'" --health-interval 10s --health-timeout 5s --health-retries 5
+        options: >-
+          --health-cmd "timeout 5 bash -c 'cat < /dev/null > /dev/udp/127.0.0.1/11211'" 
+          --health-interval 10s 
+          --health-timeout 5s 
+          --health-retries 5
+
     strategy:
       fail-fast: false
       matrix:
-        multiverse: [agent, api, background, background_2, database, httpclients, rails, rest, serialization, sinatra]
-        ruby-version: [ "2.0.0-p648", "2.1.10", "2.2.10", "2.3.8", "2.4.10", "2.5.8", "2.6.6", "2.7.1", "jruby-9.2.12.0" ]
+        ruby-version: [ 2.0.0-p648, 2.1.10, 2.2.10, 2.3.8, 2.4.10, 2.5.8, 2.6.6, 2.7.1, jruby-9.2.12.0 ]
+        multiverse: [ agent, api, background, background_2, database, httpclients, rails, rest, serialization, sinatra ]
```

Comment:
We should change the order of these so multiverse is before ruby version. Doing this will group the tests when running by multiverse group rather then ruby version. It makes it easier to go through visually.

Reply:
done!